src/Microsoft.AspNet.Mvc.DataAnnotations/NumericClientModelValidator.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.AspNet.Mvc.DataAnnotations;
+using Microsoft.Framework.Internal;
+
+namespace Microsoft.AspNet.Mvc.ModelBinding.Validation
+{
+    public class NumericClientModelValidator : IClientModelValidator
+    {
+        public IEnumerable<ModelClientValidationRule> GetClientValidationRules([NotNull] ClientModelValidationContext context)
```

Comment:
Not your code but it's strange the `[NotNull]` isn't in the interface, where the [guidelines](https://github.com/aspnet/Home/wiki/Engineering-guidelines#argument-null-checking-in-interface-member-definitions-and-abstractvirtual-methods) say it should be. Wouldn't be needed here if that were fixed.